compiler/src/iree/compiler/Codegen/LLVMCPU/ConvertToLLVM.cpp

```diff
@@ -52,6 +52,8 @@
 namespace mlir {
 namespace iree_compiler {
 
+extern llvm::cl::opt<bool> clEnableReassociateFpReductions;
```

Comment:
I didnt realize we could do that. Its interesting, but maybe we can add a boolean option flag to the `ConvertToLLVM` pass and set that by passing it through `LLVMCPU/Passes.cpp`?

Reply:
Do you mean move the definition to `ConvertToLLVM`, and the `extern` declaration to `LLVMCPU/Passes.cpp` ?

Reply:
interesting, I was not aware of this. IMO, we should add an optional flag to `ConvertToLLVM` pass, and have pass it like `createConvertToLLVMPass(clEnableReassociateFpReductions)`. Like what we've done for SplitReduction.

The Passes.cpp (i.e., pipeline constructer) has higher scope. The definition of the flag should exist in the Passes.cpp or higher layer, not in ConvertToLLVM.

Reply:
So, two separate flags? One for `SplitReduction`, and one for `ConvertToLLVM` ? But the intent of both the passes is the same, right - reorderFpReductions.

Reply:
> we should add an optional flag to ConvertToLLVM pass, and have pass it like createConvertToLLVMPass(clEnableReassociateFpReductions).

I meant the optional flag in td file, which implies that an extra argument to `createConvertToLLVMPass`.

Reply:
> > we should add an optional flag to ConvertToLLVM pass, and have pass it like createConvertToLLVMPass(clEnableReassociateFpReductions).
> 
> I meant the optional flag in td file (i.e, the ConvertToLLVM pass).

Done

Reply:
LGTM thanks!

Reply:
Can I get an LGTM for it, so that I can submit it?